react/Modal/index.jsx

```diff
@@ -3,57 +3,98 @@ import classNames from 'classnames'
 
 import styles from './index.styl'
 
-const Modal = ({
-  title, description,
-  cancelType, cancelText, cancelAction,
-  validateType, validateText, validateAction, children
-}) => (
-  <div className={styles['coz-modal-container']}>
-    <div className={styles['coz-overlay']}>
-      <div className={styles['coz-modal']}>
-        <h2 className={styles['coz-modal-title']}>{title}</h2>
-        <button
-          className={classNames('coz-btn', 'coz-btn--close', styles['coz-btn-modal-close'])}
-          onClick={cancelAction}
-          >
-          <span className='coz-hidden'>{cancelText}</span>
-        </button>
-        { description && <div className={classNames(styles['coz-modal-content'], styles['coz-description'])}>
-            {description}
-          </div>
+const ModalTitle = ({ title }) =>
+  (
+    <h2 className={styles['coz-modal-title']}>{title}</h2>
+  )
+
+const ModalCross = ({ withCross, secondaryAction, secondaryText }) =>
+  withCross &&
+  (
+    <button
+      className={classNames('coz-btn', 'coz-btn--close', styles['coz-btn-modal-close'])}
+      onClick={secondaryAction}
```

Comment:
Let's hope we'll never have to bind different actions to the cross and the secondary button. But I think this is good as it is.